docs/user-manual/modules/ROOT/pages/faq/how-does-camel-compare-to-mule.adoc

```diff
@@ -25,5 +25,5 @@ routing/mediation engines. The main differences are as follows:
 * Camel supports an implicit xref:type-converter.adoc[Type Converter] in
   the core API to make it simpler to connect components together requiring
   different types of payload & headers
-* Camel uses the https://github.com/apache/camel/blob/master/docs/user-manual/en/notice.md[Apache 2 License] rather than Mule's more restrictive
+* Camel uses the https://github.com/apache/camel/blob/master/docs/user-manual/modules/ROOT/pages/notice.md[Apache 2 License] rather than Mule's more restrictive
```

Comment:
Let's point to the official one

```suggestion
* Camel uses the https://apache.org/licenses/LICENSE-2.0[Apache 2 License] rather than Mule's more restrictive
```